src/data_viewer.rs

```diff
@@ -123,13 +123,13 @@ impl DataViewer {
                         ));
 
                         ui.menu_button("...", |ui| {
-                            ui.checkbox(&mut self.i8, "s8");
+                            ui.checkbox(&mut self.i8, "i8");
```

Comment:
Even though "i" is how rust refers to these, the decomp community tends to use the "s" and "u" prefixes, so I'd prefer to keep them the way they were

Reply:
By leaving them as-is, this will make them inconsistent, due to the change in `DataViewer` [here](https://github.com/ethteck/bdiff/pull/45).
Should we revert the `DataViewer` to keep it with `s`?

Reply:
I think I'd prefer that to having it be "i" externally. the s is synonymous with i in rust land, I assume?

Reply:
That is indeed the case.
- `i` standing for `integer` of the size specified after it.
- `u` standing for `unsigned integer` of the size specified after it

I'll revert the changes then.